TESTS/dev_mgmt/connect/main.cpp

```diff
@@ -135,51 +133,20 @@ void spdmc_testsuite_connect(void) {
 
     test_case_finish("Connect to " TEST_NETWORK_TYPE, iteration + (net_status == 0), (net_status != 0));
 
-    test_case_start("Initialize " TEST_BLOCK_DEVICE_TYPE "+" TEST_FILESYSTEM_TYPE, 2);
+    test_case_start("Initialize Storage", 2);
+
     logger("[INFO] Attempting to initialize storage.\r\n");
 
     // Default storage definition.
     BlockDevice* bd = BlockDevice::get_default_instance();
-    SlicingBlockDevice sd(bd, 0, MBED_CONF_APP_BASICS_TEST_FS_SIZE);
-#if TEST_USE_FILESYSTEM == FAT
-    FATFileSystem fs("fs", &sd);
-#else
-    LittleFileSystem fs("fs", &sd);
-#endif
-
-    test_case_finish("Initialize " TEST_BLOCK_DEVICE_TYPE "+" TEST_FILESYSTEM_TYPE, iteration + 1, 0);
-
-    if (iteration == 0) {
-        test_case_start("Format " TEST_FILESYSTEM_TYPE, 3);
-        logger("[INFO] Resetting storage to a clean state for test.\n");
-
-        int storage_status = fs.reformat(&sd);
-        if (storage_status != 0) {
-            storage_status = sd.erase(0, sd.size());
```

Comment:
The reason why I introduced the formatting in this function, is because quite often we get test failures due to the filesystem not being correctly erased. This code ensures that that we don't run into issues unrelated to the functionality of the board, mbed OS and client, i.e. the storage might have various garbage data I don't want us to test against.